modules/custom_operations/user_ie_extensions/tokenizer/python/ov_tokenizer/utils.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+from typing import Dict, Optional, Sequence, Tuple, Union
+
+from constants import LOGITS_OUTPUT_NAME, TOKEN_IDS_OUTPUT_NAME
```

Comment:
```suggestion
from .constants import LOGITS_OUTPUT_NAME, TOKEN_IDS_OUTPUT_NAME
```
It fails to import otherwise after `py -m pip install .[transformers]`